### PR TITLE
[SIG-3443] Convert the MultiTextInput to typescript

### PR DESCRIPTION
**Pull request by aditudorache**
This PR converts the MultiTextInput wizard control to `typescript`

### CODE REVIEW

File: src/signals/incident/components/form/MultiTextInput/MultiTextInput.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import type { FunctionComponent } from 'react';
+import React from 'react';
+import styled from 'styled-components';
+import map from 'lodash.map';
+
+import Input from 'components/Input';
+import Button from 'components/Button';
+
+import './style.scss';
```
Comment (MauritsR) [inline src/signals/incident/components/form/MultiTextInput/MultiTextInput.tsx:9]: Can we take this opportunity to remove the scss file? It only applies a margin-bottom styling

File: src/signals/incident/components/form/MultiTextInput/MultiTextInput.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+const allowedChars = /[\d,.;]+/;
+
+const filterInvalidKeys = (event: React.KeyboardEvent<HTMLInputElement>) => {
+  if (!allowedChars.test(event.key)) {
+    // Swallow invalid character inputs.
+    event.preventDefault();
+  }
+};
+
+interface ParentType { meta: { updateIncident: (data: any) => void }}
```
Comment (MauritsR) [inline src/signals/incident/components/form/MultiTextInput/MultiTextInput.tsx:23]: This type is implemented in https://github.com/Amsterdam/signals-frontend/pull/1444, can you reuse that?

File: src/signals/incident/components/form/MultiTextInput/MultiTextInput.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  if (!fields.length) {
+    fields.push('');
+  }
+
+  fields.push('');
+  parent.meta.updateIncident({ [meta.name]: fields });
+}
+
+export const StyledInput = styled(Input)`
+  margin-bottom: 8px;
```
Comment (MauritsR) [inline src/signals/incident/components/form/MultiTextInput/MultiTextInput.tsx:50]: themeSpacing?

**Review by MauritsR — CHANGES_REQUESTED** [review]

File: src/signals/incident/components/form/MultiTextInput/MultiTextInput.tsx (hunk 1 above)
Comment (aditudorache) [reply src/signals/incident/components/form/MultiTextInput/MultiTextInput.tsx:9]: Sure thing!!

**Review by MauritsR — APPROVED** [review]

**Pull request merged**